tools/accuracy_checker/accuracy_checker/main.py

```diff
@@ -37,147 +37,279 @@
+            type=cast_to_bool,
+            required=False
+        )
+        common_args.add_argument(
+            '--input_precision',
+            help='space-separated list of precisions for network inputs. '
+                 'Providing several values required <layer_name>:<precision> format. '
+                 'If single value without layer_name provided, then it will be applied to all input layers.',
+            required=False,
+            nargs='+'
+        )
 
-    parser.add_argument(
-        '-tt', '--target_tags',
-        help='Space separated list of launcher tags for infer',
-        required=False,
-        nargs='+'
-    )
+    def add_dataset_related_args(parser):
+        dataset_related_args = parser.add_argument_group('Datasets-related arguments')
+        dataset_related_args.add_argument(
+            '-ss', '--subsample_size',
+            help="dataset subsample size",
+            type=str,
+            required=False
+        )
+        dataset_related_args.add_argument(
+            '--shuffle',
+            help="allow shuffle annotation during creation a subset",
+            type=cast_to_bool,
+            required=False
+        )
+        dataset_related_args.add_argument(
+            '--store_subset',
+            help='allow to save evaluation data ids',
+            type=cast_to_bool,
+            default=False,
+            required=False
+        )
+        dataset_related_args.add_argument(
+            '--subset_file',
+            help='file name for saving or reading identifiers subset',
+            required=False
+        )
 
-    parser.add_argument(
-        '-l', '--log_file',
-        help='file for additional logging results',
-        required=False
-    )
+    def add_profiling_related_args(parser):
+        profiling_related_args = parser.add_argument_group('Profiling related arguments')
+        profiling_related_args.add_argument(
+            '--profile',
+            help='activate metric profiling mode',
+            type=cast_to_bool,
+            required=False
+        )
+        profiling_related_args.add_argument(
+            '--profiler_logs_dir',
+            help='path to save profiler logs',
+            type=partial(get_path, is_directory=True),
+            default=Path.cwd(),
+            required=False
+        )
+        profiling_related_args.add_argument(
+            '--profile_report_type',
+            help='report type for profiler logs',
+            default='csv',
+            choices=['csv', 'json'],
+            required=False
+        )
 
-    parser.add_argument(
-        '--ignore_result_formatting',
-        help='allow to get raw metrics results without data formatting',
-        required=False,
-        default=False,
-        type=cast_to_bool
-    )
+    def add_tool_customization_args(parser):
+        tool_customization_args = parser.add_argument_group('Tool customization arguments')
```

Comment:
for me castomization means that you modify running tool, but we just setup some logging (results, progress and so on)
better
```suggestion
        tool_customization_args = parser.add_argument_group('Tool settings arguments')
```
or logging arguments
BTW, log file should be also in this category

Reply:
Renamed